packages/web/package.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "@concordium/web",
+    "version": "0.0.1",
+    "license": "Apache-2.0",
+    "private": false,
+    "main": "lib/src/index.js",
+    "types": "lib/src/index.d.ts",
+    "files": [
+        "/lib/**/*"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Concordium/concordium-node-sdk-js"
+    },
+    "author": {
+        "name": "Concordium Software",
+        "email": "support@concordium.software",
+        "url": "https://concordium.com"
+    },
+    "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^4.28.1",
+        "@typescript-eslint/parser": "^4.28.1",
+        "babel-jest": "^27.0.6",
+        "cross-env": "5.0.5",
+        "eslint": "^7.29.0",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-import": "^2.23.4",
+        "eslint-plugin-prettier": "^3.4.0",
+        "jest": "^27.0.6",
+        "ts-jest": "^27.0.3",
+        "typescript": "^4.3.5",
+        "webpack": "^5.72.0",
+        "webpack-cli": "^4.9.2",
+        "stream-browserify": "^3.0.0"
+    },
+    "scripts": {
+        "build": "tsc",
+        "postbuild": "yarn webpack",
```

Shouldn't this just be

```suggestion
        "postbuild": "webpack",
```